github-orgs/kubeflow/org.yaml

```diff
@@ -929,6 +929,7 @@ orgs:
             members:
             - Jeffwan
             - gabrielwen
```

Comment:
Please remove @gabrielwen and @zhenghuiwang

Reply:
Addressed